js/parse.js

```diff
@@ -14,6 +14,26 @@ function parseData(rawJson) {
         newCharacter.name = charNameData[i];
         characters.push(newCharacter);
     }
+
+    const fieldsPrint = JSON.parse(fields.Print.stringValue);
+    const printData = fieldsPrint.slice(5, fieldsPrint.length); // REMOVE 5 '0' ELEMENTS
+    const chunk = 14; // # OF SAMPLE SLOTS OF ONE CHARACTER
+    let charsIterator = 0;
+    for (var z = 0; charsIterator < 9; z += chunk) {
+        let tempArray = printData.slice(z, z + chunk);
+        characters[charsIterator] = {
+            ...characters[charsIterator],
+            printer: [
+                {
+                    item: itemMap[tempArray[10]],
+                    amount: tempArray[11]
+                },
+                { item: itemMap[tempArray[12]], amount: tempArray[13] }
+            ]
+        };
+        charsIterator++;
+    }
```

Comment:
All of this new code should be moved inside the fillCharacterData function. 

There, you will be able to use the index "i" instead of making a whole new loop. 

I also suggest you put it at the bottom of the function (right under equipped bubbles), as that's what I've been doing with new stuff.